.github/ISSUE_TEMPLATE/config.yml

```
@@ -4,8 +4,8 @@ contact_links:
     url: https://rucio.cern.ch/documentation/
     about: The Rucio Documentation might give you some information related to the problem.
   - name: ❓ Question - Ask us
-    url: https://rucio.cern.ch/contact.html
-    about: This issue tracker is not for technical support. Please use one of our dedicated channels, and ask the community for help.
+    url: https://rucio.cern.ch/documentation/contact_us/
+    about: The contact us webpage will point you to the communication channel best suited to your questions.
```

I would stay along the lines of the old `about` here. This is the issue template that appears when you open a new issue in the repository, so it makes sense to mention to the user that they should not open an issue for technical support, and they should instead use one of the contact channels.

```suggestion
    about: If you do not need to report a bug or request a feature, but you have a question or need technical support, please use one of our contact channels listed here.
```

Or "If you have a question or need technical support, please use one of our contact channels listed here, instead of opening an issue."